### DISCUSSION TITLE
Agenda and tasks together [android]

### DISCUSSION THREAD

**Discussion by bepolymathe** [TechbeeAT/jtxBoard#620] [General]
Hi @patrickunterwegs 

I don't know about you, but I really need to see my tasks and my agenda at the same time. My tasks and notes are managed with jtx but not my pro/perso agenda. On the desktop, no worries because I can display everything together in Thunderbird. 

However, with Android it seems more complicated. Is there an Android calendar application that displays jtx tasks (synchronized via Nextcloud in my case)? Is there any prospect of integrating the agenda into jtx  in the futur ?

**Comment by patrickunterwegs** [comment]
Hey @bepolymathe , 
There is no app that shows jtx Board tasks and calendar entries. Integrating the Android calendar is on the roadmap though!

**Reply by bepolymathe** [reply]
I'll waiting for that ! Thanks again for this super app.

**Comment by Samanbaia** [comment]
Dear @patrickunterwegs,
I personally would "love" to have the journals entries as "appointments" in my (android or linux/thunderbird) calendar. The idea would be then to participate in a meeting and write information regarding the meeting, then add tasks to do regarding this appointment. I would be then easy to go back in time and find any information regarding this meeting. On the other hand, I would then propose a future appointment (future journal), and start placing task to do in order to prepare this future meeting in my agenda.
To react to @bepolymathe , I use TODO (in FDROID) that was showing the tasks in the widget, but it was synchronizing separately the tasks and the agenda, so was showing them in the widget. With JTX, you just sync JTX, and shows only the agenda entries. Maybe is there a workaround, if you manage to duplicate the tasks somewhere.

**Reply by patrickunterwegs** [reply]
Hey @Samanbaia , sure, I will consider that. It's just work that needs to be done some time. Currently I'm lacking a bit of motivation after a few unnecessary negative comments on the Google Play Store, but in the future I plan to add the calendar integration step by step. 
I'm not sure if I'd ever make it the other way round, meaning to show journals and tasks in the local calendar. This would be a bit tough to keep entries in sync because the local calendar storage in Android only allows VEVENTs (and no VTODOs or VJOURNALs)

**Reply by rogercreagh** [reply]
When I go to play store I see jtxB listed with no reviews and no comments. As far as I'm concerned jtxB is possibly the most useful app on my phone and tablet - its certainly one that I use every day without problems.
You are also excellent at responding and sorting out issues, or taking the time to explain why they are not issues.

You do seem to be spending a mammoth amount of energy on maintenance and bug fixing and minor improvements.  The core app is solid, perhaps you need to give a bit more time and attention to yourself. I'd have thought negative comments on Google are as likely to be from shills for some commercial organisation thinking of entering the space jtxB has opened up as serious considered and useful criticism/suggestions. Negativity sucks life out, try and ignore it, you're doing good.

RogerCO

PS what about issue #645 (which covers a lot of non-urgent but all related niggling items/suggestions) - but only when you've got your mojo back :wink:

**Comment by Samanbaia** [comment]
I am sorry to hear regarding the bad comments, and I don't know how to support you more than telling you your work helps me to organize my life and my job. I would love to give a hand in coding, but I am too old for that now, and I am not any more smart enough.

Furthermore, I didn't know the limitation regarding the implementation of VTODOs and VJOURNALs in ANDROID agenda: as told to @bepolymathe, I work with TODO as widget that shows VTODOs. That can be an idea also for VJOURNALs? Obviously, it is just an idea and I don't know if it is worth the value: for me, I dont' know how your DB is structured, but if I imagine that it is based on a timeline where the time is the creation of the JOURNAL/NOTE/TASK, then, the agenda would just be a visual interface of this timeline. That would be lovely... and I imagine then it would be easier to have it in Thunderbird (maybe...)

Good luck for everything,
